foundation_cms/blocks/pillar_card_set_block.py

```diff
@@ -0,0 +1,24 @@
+from wagtail import blocks
+
+from foundation_cms.base.models.base_block import BaseBlock
+
+from . import OptionalLinkBlock, PillarCardBlock
+
+NUM_PILLAR_CARDS = 3
+DEFAULT_PILLAR_CARDS = [{"headline": "", "image": None, "cta_link": []} for _ in range(NUM_PILLAR_CARDS)]
```

Comment:
Thanks @mmmavis for this great idea. Now when a user clicks on this block for use, there will be 3 blank cards ready to go, instead of being greeted with 1 and a red validation error mentioning that the minimum number is 3.